parse.go

```diff
@@ -524,3 +524,58 @@ func TCFVersionFromTCString(s string) TCFVersion {
 		return InvalidTCFVersion
 	}
 }
+
+// ParseUsNational takes a base64 Raw URL Encoded string which represents a US National
+// consent string according to the IAB's Multi-state Privacy Agreement and returns a
+// MspaParsedConsent with its fields populated with the values stored in the string.
+//
+// Example Usage:
+//
+//   var pc, err = iabconsent.ParseUsNational("BVVqAAEABAA.QA")
+func ParseUsNational(s string) (GppParsedConsent, error) {
+	var segments = strings.Split(s, ".")
```

Comment:
Do we still need this function? if so can we do a NewMspaNationl(s) and reuse the *MspaUsNational.ParseConsent() function!

Reply:
Good catch, was going to remove this once I transferred to the interface! Removed.